edgepi-adc.html

```diff
@@ -0,0 +1,248 @@
+<script type="text/javascript">
+	RED.nodes.registerType('edgepi-adc-node',{
+		category: 'Osensa EdgePi',
+		color: '#a6bbcf',
+		defaults: {
+		name: {value: ""},
+    transport: { value: "Local" },
+    tcpAddress: { value: "" },
+    tcpPort: { value: "" },
+		read: { value: "voltage" },
+    adc: { value: "ADC_1"},
+    adc1DataRate: { value: "SPS_38400"},
+    adc2DataRate: { value: "SPS_800"},
+		channel: { value: "AIN1"},
+    diff: { value: "DIFF_1"},
+  	},
+		inputs:1,
+		outputs:1,
+	  icon: "right-arrow-bracket.svg",
+		label: function() {
+			return this.name||"adc-node";
+    },
+    oneditprepare: function(){
+      // Query html elements
+      const configTransportSelect = document.getElementById("node-input-transport");
+      const tcpField = document.querySelector(".form-row.tcp");
+			const read = document.getElementById("node-input-read");
+      const adcOptions = document.getElementById("node-input-adc");
+			const adc1 = document.getElementById("adc-option-adc1");
+      const dataRateField = document.querySelector(".form-row.data-rate")
+      const adc1DataRate = document.getElementById("node-input-adc1DataRate");
+      const adc2DataRate = document.getElementById("node-input-adc2DataRate");
+			const channelField = document.querySelector(".form-row.channel");
+      const diffField = document.querySelector(".form-row.diff");
+
+      function updateEditor(){
+				tcpField.style.display = (configTransportSelect.value === "Network") ? "flex" : "none";
+				channelField.style.display = (read.value === "voltage") ? "flex" : "none";
+        dataRateField.style.display = (read.value === "voltage") ? "flex": "none";
+        diffField.style.display = (read.value === "differential") ? "flex" : "none";
+        adc1DataRate.style.display = (adcOptions.value === "ADC_1") ? "flex" : "none";
+        adc2DataRate.style.display = (adcOptions.value === "ADC_2") ? "flex" : "none";
```

Comment:
What is the point of removing them from the page? They can still be interacted with through the DOM but I also don't really know how Node-RED works.

Reply:
I'm not sure why someone would want to interact with it through the DOM lol.. but you're right I guess they could. Is what I'm doing bad practice?

Reply:
Not sure honestly. I guess if they are manually editing the DOM and messing with the styling/UI then the user must know what they are doing.